mbzirc_ign/launch/spawn.launch.py

```diff
@@ -82,6 +82,14 @@ def spawn(context, model_type, world_name, model_name, position):
         bridges.extend(payload_bridges)
         nodes.extend(payload_nodes)
 
+    if model.isFixedWingUAV():
+        nodes.append(Node(
+            package='mbzirc_ros',
+            executable='fixed_wing_bridge',
+            output='screen',
+            parameters=[{'model_name': model_name}],
+        ))
```

Comment:
we'll also need to add this to `spawn_config.launch.py` around [here](https://github.com/osrf/mbzirc/blob/main/mbzirc_ign/launch/spawn_config.launch.py#L53)